eternalcore-core/src/main/java/com/eternalcode/core/configuration/migration/impl/PC1.java

```diff
@@ -0,0 +1,23 @@
+package com.eternalcode.core.configuration.migration.impl;
+
+import com.eternalcode.core.configuration.migration.AbstractMigration;
+
+import static com.eternalcode.core.configuration.migration.MigrationStep.move;
+
+public class PC1 extends AbstractMigration {
```

Comment:
Are you sure that this name isn’t too generic? We should use something more understandable.